docs/release-notes/1.0.rst

```diff
@@ -1,51 +0,0 @@
-1.0 Release Notes
```

Comment:
Same for these. I think we will need to bring the release notes back at some point, but it's fine if they go for now.